pkg/apis/sources/v1alpha1/apiserver_conversion_test.go

```diff
@@ -34,20 +36,8 @@ import (
 	duckv1 "knative.dev/pkg/apis/duck/v1"
 )
 
-// TODO: Replace dummy some other Eventing object once they
-// implement apis.Convertible
-type dummy struct{}
-
-func (*dummy) ConvertTo(ctx context.Context, obj apis.Convertible) error {
-	return errors.New("Won't go")
-}
-
-func (*dummy) ConvertFrom(ctx context.Context, obj apis.Convertible) error {
-	return errors.New("Won't go")
-}
-
```

Comment:
I think I'd rather duplicate this all over, otherwise the test_helper might be included in the binary...

Reply:
agreed offline that we will duplicate for now